packages/next/export/worker.ts

```diff
@@ -208,6 +210,14 @@ export default async function exportPage({
         req.url += '/'
       }
 
+      if (
+        buildExport &&
+        renderOpts.domainLocales &&
+        renderOpts.domainLocales.length > 0
```

Comment:
Should this be checking if the current path being rendered matches a domain locale instead of just setting this to always true if there are any domain locales configured?